google/cloud/forseti/services/dao.py

```diff
@@ -478,7 +478,8 @@ class Permission(base):
         """Row entry for an IAM permission."""
 
         __tablename__ = permissions_tablename
-        name = Column(String(256), primary_key=True)
+        name = Column(get_string_by_dialect(dbengine.dialect.name, 256),
```

Comment:
Main change to fix the duplicate permission. Other columns in the model tables do this as well.